docs/source/_autogenerated/explorer_defaults.rst

```diff
@@ -43,276 +43,276 @@ chain_id = 137
-multicall2 = ""
+multicall2 = "None"
 chain_id = 1088
 
 [polygon-zkevm]
 explorer = "https://zkevm.blockscout.com/"
 explorer_type = "blockscout"
-multicall2 = ""
+multicall2 = "None"
 chain_id = 1101
 
 [core]
-explorer = ""
-multicall2 = ""
+explorer = "None"
+multicall2 = "None"
 chain_id = 1116
 
 [lisk]
 explorer = "https://blockscout.lisk.com/"
 explorer_type = "blockscout"
-multicall2 = ""
+multicall2 = "None"
 chain_id = 1135
 
 [reya-network]
 explorer = "https://explorer.reya.network/"
 explorer_type = "blockscout"
-multicall2 = ""
+multicall2 = "None"
 chain_id = 1729
 
 [onus]
 explorer = "https://explorer.onuschain.io/"
 explorer_type = "blockscout"
-multicall2 = ""
+multicall2 = "None"
 chain_id = 1975
 
 [hubblenet]
-explorer = ""
-multicall2 = ""
+explorer = "None"
+multicall2 = "None"
 chain_id = 1992
 
 [sanko]
 explorer = "https://explorer.sanko.xyz/"
 explorer_type = "blockscout"
-multicall2 = ""
+multicall2 = "None"
 chain_id = 1996
 
 [dogechain]
 explorer = "https://explorer.dogechain.dog/"
 explorer_type = "blockscout"
-multicall2 = ""
+multicall2 = "None"
 chain_id = 2000
 
 [milkomeda]
 explorer = "https://explorer-mainnet-cardano-evm.c1.milkomeda.com/"
 explorer_type = "blockscout"
-multicall2 = ""
+multicall2 = "None"
 chain_id = 2001
 
 [kava]
 explorer = "https://testnet.kavascan.com/"
 explorer_type = "blockscout"
-multicall2 = ""
+multicall2 = "None"
 chain_id = 2222
 
 [mantle]
 explorer = "https://explorer.mantle.xyz/"
 explorer_type = "blockscout"
-multicall2 = ""
+multicall2 = "None"
 chain_id = 5000
 
 [zetachain]
 explorer = "https://zetachain.blockscout.com/"
 explorer_type = "blockscout"
-multicall2 = ""
+multicall2 = "None"
 chain_id = 7000
 
 [celo]
 explorer = "https://explorer.celo.org/mainnet/"
 explorer_type = "blockscout"
-multicall2 = ""
+multicall2 = "None"
 chain_id = 42220
 
 [oasis]
 explorer = "https://explorer.emerald.oasis.dev/"
 explorer_type = "blockscout"
-multicall2 = ""
+multicall2 = "None"
 chain_id = 42262
 
 [linea]
 explorer = "https://explorer.linea.build/"
 explorer_type = "blockscout"
-multicall2 = ""
+multicall2 = "None"
 chain_id = 59144
 
 [blast]
 explorer = "https://blast.blockscout.com/"
 explorer_type = "blockscout"
-multicall2 = ""
+multicall2 = "None"
 chain_id = 81457
 
 [taiko]
 explorer = "https://blockscoutapi.mainnet.taiko.xyz/"
 explorer_type = "blockscout"
-multicall2 = ""
+multicall2 = "None"
 chain_id = 167000
 
 [scroll]
 explorer = "https://blockscout.scroll.io/"
 explorer_type = "blockscout"
-multicall2 = ""
+multicall2 = "None"
 chain_id = 534352
 
 [zora]
 explorer = "https://explorer.zora.energy/"
 explorer_type = "blockscout"
-multicall2 = ""
+multicall2 = "None"
 chain_id = 7777777
 
 [neon]
 explorer = "https://neon.blockscout.com/"
 explorer_type = "blockscout"
-multicall2 = ""
+multicall2 = "None"
 chain_id = 245022934
 
 [aurora]
 explorer = "https://explorer.mainnet.aurora.dev/"
 explorer_type = "blockscout"
-multicall2 = ""
+multicall2 = "None"
```

Comment:
https://github.com/toml-lang/toml/issues/30 :)

Reply:
ew. I don't like how they just say "if a key is none, just don't leave it". ug.